packages/@lwc/synthetic-shadow/src/faux-shadow/traverse.ts

```diff
@@ -38,18 +38,20 @@ function foldSlotElement(slot: HTMLElement) {
 
 function isNodeSlotted(host: Element, node: Node): boolean {
     if (process.env.NODE_ENV !== 'production') {
-        assert.invariant(
-            host instanceof HTMLElement,
-            `isNodeSlotted() should be called with a host as the first argument instead of ${host}`
-        );
-        assert.invariant(
-            node instanceof Node,
-            `isNodeSlotted() should be called with a node as the second argument instead of ${node}`
-        );
-        assert.invariant(
-            compareDocumentPosition.call(node, host) & DOCUMENT_POSITION_CONTAINS,
-            `isNodeSlotted() should never be called with a node that is not a child node of ${host}`
-        );
+        if (!(host instanceof HTMLElement)) {
+            // eslint-disable-next-line no-console
+            console.error(`isNodeSlotted() should be called with a host as the first argument`);
+        }
+        if (!(node instanceof Node)) {
+            // eslint-disable-next-line no-console
+            console.error(`isNodeSlotted() should be called with a node as the second argument`);
+        }
+        if (!(compareDocumentPosition.call(node, host) & DOCUMENT_POSITION_CONTAINS)) {
+            // eslint-disable-next-line no-console
+            console.error(
+                `isNodeSlotted() should never be called with a node that is not a child node of the given host`
+            );
+        }
```

Comment:
I'm guessing the original intention of these invariants was "This will never ever happen, so fail early in dev mode so we (LWC) can figure out where our implementation is wrong." Synthetic shadow is in maintenance mode, so that strategy doesn't really make much sense anymore, especially if component authors are actually able to hit these lines.